OWNERS

```diff
@@ -3,3 +3,5 @@
 approvers:
 - mikenairn
 - maleck13
+- laurafitzgerald
+- StevenTobin
```

Comment:
@laurafitzgerald @StevenTobin Adding you both here, this is mostly to have more options for approvers on the CI PRs since they get synced with this file.